bindings/py/nupic/bindings/regions/TestNode.py

```diff
@@ -299,8 +302,11 @@ def getProtoType():
 
 
   def write(self, proto):
-    pass
-
+    regionImpl = proto.regionImpl.as_struct(TestNodeProto)
+    regionImpl.int32Param = self.getParameter('int32Param', 0)
```

Comment:
No reason, just that it's extra work especially since some of the parameters are lists, and it doesn't tell us more about whether this is working correctly or not. If you think it's worth including in this PR, let me know and I can do it.